frontend/app-development/features/appPublish/components/Deploy/DeployDropdown/DeployDropdown.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { type ReactElement } from 'react';
+import classes from './DeployDropdown.module.css';
+import { StudioCombobox, StudioError, StudioSpinner } from '@studio/components';
+import type { ImageOption } from '../../ImageOption';
+import { useTranslation } from 'react-i18next';
+import { useAppReleasesQuery } from 'app-development/hooks/queries';
+import { BuildResult } from 'app-shared/types/Build';
+import { DateUtils } from '@studio/pure-functions';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { DeployPopover } from './DeployPopover';
+
+export type DeployDropdownProps = {
+  appDeployedVersion: string;
+  disabled: boolean;
+  setSelectedImageTag: (tag: string) => void;
+  selectedImageTag: string;
+  startDeploy: () => void;
+  isPending: boolean;
+};
+
+export const DeployDropdown = ({
+  appDeployedVersion,
+  selectedImageTag,
+  setSelectedImageTag,
+  disabled,
+  startDeploy,
+  isPending,
+}: DeployDropdownProps): ReactElement => {
+  const { org, app } = useStudioEnvironmentParams();
+  const { t } = useTranslation();
+
+  const {
+    data: releases = [],
+    isPending: releasesIsPending,
+    isError: releasesIsError,
+  } = useAppReleasesQuery(org, app, { hideDefaultError: true });
+
+  if (releasesIsPending)
```

Comment:
I think readability improves by changing the spelling:
```suggestion
  if (isPendingReleases)
```